src/js/script.js

```diff
@@ -0,0 +1,63 @@
+let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, ''];
+let indexOfVoid = 15;
+let winCombination1 = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
+let winCombination2 = [1, 5, 9, 13, 2, 6, 10, 14, 3, 7, 11, 15, 4, 8, 12];
+
+function shuffle() {
+    let mySet = new Set();
```

Comment:
Bad naming. Why just 'mySet'?
What the reason to use set here? To be sure that numbers will be unique?
Lets look how many iterations you will get below